src/python/engagement-creator/engagement_creator/BUILD

```diff
@@ -1 +1,9 @@
-python_library()
\ No newline at end of file
+python_library()
+
+pex_binary(
+    name="pex",
+    output_path="engagement-creator.pex",
+    entry_point="./run.py:run",
+    execution_mode="venv",
+)
+
```

Comment:
actual-new-change-from-this-pr

this is copying the pattern jesse introduced in analyzer-executor/BUILD